ssh/tailssh/incubator.go

```diff
@@ -233,6 +242,19 @@ func beIncubator(args []string) error {
 	return cmd.Run()
 }
 
+// subprocEnviron filters and returns subset of os.Environ.
+func subprocEnviron() []string {
+	all := os.Environ()
+	ret := all[:0]
+	for _, e := range all {
+		if strings.HasPrefix(e, "PATH=") {
+			continue
```

Comment:
Why do we remove PATH? This just makes me wonder what else is in this environment that needs to be nuked. Is tailscaled setting a restricted env for the incubator?

Reply:
Ah, I see below that it's kind of so that the incubator can find stuff like login. Hmm. At least add a comment here explaining why PATH is special? But also I'm wondering if we can do all those path lookups in the parent and keep the incubator less surprising.

Reply:
I removed the PATH and instead just passed the path for `login` as a flag